docs/doobie.md

```diff
@@ -0,0 +1,40 @@
+# Doobie
+
+[![Maven Central](https://img.shields.io/maven-central/v/com.avast/sst-doobie-hikari_2.12)](https://repo1.maven.org/maven2/com/avast/sst-doobie-hikari_2.12/)
+
+`libraryDependencies += "com.avast" %% "sst-doobie-hikari" % "<VERSION>"`
+
+This module initializes a doobie `Transactor`:
+
+```scala
+import cats.effect.Resource
+import com.avast.sst.doobie.DoobieHikariModule
+import com.avast.sst.example.config.Configuration
+import com.avast.sst.jvm.execution.ConfigurableThreadFactory.Config
+import com.avast.sst.jvm.execution.{ConfigurableThreadFactory, ExecutorModule}
+import com.avast.sst.micrometer.jmx.MicrometerJmxModule
+import com.avast.sst.pureconfig.PureConfigModule
+import com.zaxxer.hikari.metrics.micrometer.MicrometerMetricsTrackerFactory
+import scala.concurrent.ExecutionContext
+import zio._
+import zio.interop.catz._
+import zio.interop.catz.implicits._
+
+implicit val runtime = new DefaultRuntime {} // this is just needed in example
+
+for {
+  configuration <- Resource.liftF(PureConfigModule.makeOrRaise[Task, Configuration])
+  executorModule <- ExecutorModule.makeFromExecutionContext[Task](runtime.Platform.executor.asEC)
+  meterRegistry <- MicrometerJmxModule.make[Task](configuration.jmx)
+  boundedConnectExecutionContext <- executorModule
+                                     .makeThreadPoolExecutor(
+                                       configuration.boundedConnectExecutor,
+                                       new ConfigurableThreadFactory(Config(Some("hikari-connect-%02d")))
+                                     )
+                                     .map(ExecutionContext.fromExecutorService)
+  hikariMetricsFactory = new MicrometerMetricsTrackerFactory(meterRegistry)
+  doobieTransactor <- DoobieHikariModule
```

Comment:
can you pls provide some documentation why hikari was chosen? I never heard about this

Reply:
I think it's the industry standard for SQL connection pool, not much to "document". Am I right @sideeffffect?

Reply:
i doubt it's `industry standart` :D but im not against it - just would like to see at least some rationalization since i never heard about this library nor have ever seen it used

Reply:
It's the only connection pool that's documented on Dobbie's website, so I guess vast majority of Doobie users use Hikari :laughing: 
https://tpolecat.github.io/doobie/docs/14-Managing-Connections.html#using-a-hikaricp-connection-pool

Reply:
How can you doubt it when you don't know anything about it? I think you get mislead by the name which seems strange but it really is a very popular JDBC connection pooling library. Sure, frameworks like Spring or J2EE have probably their own solution but in the J2SE-land Hikari is definitely in the top 3. Second link from Google search mentions it: https://www.baeldung.com/java-connection-pooling Another argument which @sideeffffect told me is that doobie does not support anything else so there's not much choice ;) But I guess author of doobie chose it for some reason, right?

Reply:
ok, fair enough ... it seemd like we are using some opt-in functionality ... do we need to call it `doobiehikari` module then? could we call it just `DoobieModule`?

Reply:
Well, I look into the future and expect that there could be more implementations. This is consistent with the way `http4s` modules are structured etc.

Reply:
i suspected that is the case - i generally dislike polliuting api with vision that there can be some different implementation eventually (even though it's been years and non came up and the default is *industry standart*) i think it's a case of premature optimization ... but we probably not gonna find an agreement on this issue so do what you wish